pkgs/by-name/bu/bulk_extractor/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, exiv2
+, flex
+, libewf
+, libxml2
+, openssl
+, tre
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bulk_extractor";
+  version = "2.0.6";
+
+  src = fetchFromGitHub {
+    owner = "simsong";
+    repo = "bulk_extractor";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-LNdRN4pEA0rVEyKiBKGJgTKA4veVvsuP3ufiolHTk/s=";
+    fetchSubmodules = true;
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ exiv2 flex libewf libxml2 openssl tre zlib ];
+
+  meta = with lib; {
+    description = "A program for scanning and extracting useful information";
+    longDescription = ''
+      bulk_extractor is a C++ program that scans a disk image, a file, or a directory of files and extracts useful information without parsing the file system or file system structures. The results are stored in feature files that can be easily inspected, parsed, or processed with automated tools.
+    '';
+    mainProgram = "bulk_extractor";
+    homepage = "https://github.com/simsong/bulk_extractor";
+    downloadPage = "http://downloads.digitalcorpora.org/downloads/bulk_extractor/";
+    changelog = "https://github.com/simsong/bulk_extractor/blob/v${finalAttrs.version}/ChangeLog";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = with platforms; linux;
+    license = licenses.mit;
```

This code seems to be under all sorts of licenses, where every single file specifies the license in it's header.

Not sure if the `openssl` token will collide with the other `openssl` token from the package imports under the `with licenses` statement.

```suggestion
    license = with licenses; [
      mit
      cpl10
      gpl3Only
      lgpl21Only
      lgpl3Only
      openssl
    ];
```